### PR TITLE
Feature/google aliases

### PR DESCRIPTION
**Pull request by siepet**
It makes it possible to add and remove user aliases, using permissions/users yaml files with specified aliases array.

### CODE REVIEW

File: app/models/google_integration/actions/log.rb [hunk 1]
```diff
@@ -120,6 +122,22 @@ def log_removing_aliases
           end
         end if @diff_hash[:remove_aliases]
       end
+
+      def log_adding_user_aliases
+        @diff_hash[:add_user_aliases].each do |user, aliases|
+          aliases.each do |a|
+            @log << "[api] add user alias #{a} to user #{user.email}"
+          end
+        end if @diff_hash[:add_user_aliases]
```
Comment (lubieniebieski) [inline app/models/google_integration/actions/log.rb]: `.present?` ?

Comment (siepet) [inline app/models/google_integration/actions/log.rb]: Trying to be consistent with the rest of the code.

Comment (lubieniebieski) [inline app/models/google_integration/actions/log.rb]: be consistent with *good* patterns:)

File: spec/support/google_api.rb [hunk 2]
```diff
@@ -36,14 +36,21 @@
...
         email: 'fourth@netguru.pl',
-        primaryEmail: 'fourth@netguru.pl',
+        primaryEmail: 'member2@foo.pl',
+        aliases: ['secundo'],
+      ),
+      Hashie::Mash.new(
+        name: 'first.member1',
+        email: 'member1@foo.pl',
+        primaryEmail: 'member3@foo.pl',
+        aliases: [],
```
Comment (lubieniebieski) [inline spec/support/google_api.rb]: try your tests with this line as a `nil` - it seems to be the default behaviour for returned object

File: app/models/google_integration/actions/diff.rb [hunk 3]
```diff
@@ -100,6 +106,19 @@ def difference_membership(expected, domain, group)
         end
       end
 
+      def user_aliases_diff(google_users, dataguru_users)
+        google_users.each do |google_user|
```
Comment (lubieniebieski) [inline app/models/google_integration/actions/diff.rb]: could you please add `resque` here so we catch `UserErro` from `UserRepository` when user is not found?

**Comment by lubieniebieski** [comment]
codeclimate yells about duplication - imho it would be better to leave things as they are right now - at least we see what's going on.

**Pull request merged**